lib/correios.js

```diff
@@ -88,4 +87,47 @@ Correios.prototype.consultaCEP = function(args, callback) {
   });
 };
 
+
+Correios.prototype.validArg = function(args){
+    //decimal is number
+    var fields = {
+      nCdServico :'string',
+      sCepOrigem :'string',
+      sCepDestino :'string',
+      nVlPeso :'string',
+      nCdFormato :'number',
+      nVlComprimento :'number',
+      nVlAltura :'number',
+      nVlLargura :'number',
+      nVlDiametro:'number'
+    }
+
+
+    for(key in fields){
+      //i check if is defined, cause the user can set wrong fields, so i ignore that
+      if(args[key] != undefined){
+          //here i check if is 'envelope', seeing for other validations
+          if(key == 'nCdFormato'){
```

Comment:
Estava testando aqui e esqueci de voltar a validação certa

          if(key == 'nCdFormato' $$ args['nCdFormato'] == 3){ // só entra se for envelope